apps/seqan_flexbar/adapter_trimming.h

```diff
@@ -87,12 +87,17 @@ struct Mode {};
 struct Auto : Mode {};
 
 //Tagging struct representing the the match algorithm working
-//with values supplied by the user. Saves those values as members.
+//with values supplied by the user. Saves those  values as members.
```

Comment:
I changed it, now its not so ugly anymore :)
BTW: does seqan::StringSet have an iterator?